docs/ksh/builtins.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "https://www.w3.org/TR/REC-html40/loose.dtd">
```

Comment:
These doctype changes should not be made. Those aren't real URLs but [XML namespaces names](https://en.wikipedia.org/wiki/XML_namespace#Namespace_names).